sourmash_lib/commands.py

```diff
@@ -17,6 +17,29 @@
 WATERMARK_SIZE = 10000
 
 
+def info(args):
+    "Report sourmash version + version of installed dependencies."
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-v', '--verbose', action='store_true',
+                        help='report versions of khmer and screed')
+    args = parser.parse_args(args)
+
+    from . import VERSION
+    notify('sourmash version {}', VERSION)
+    notify('- loaded from path: {}', os.path.dirname(__file__))
+    notify('')
+
+    if args.verbose:
+        import khmer
```

Comment:
We should probably make `khmer` a [hard dependency in the project][0], or do a try/except block here.

[0]: https://github.com/dib-lab/sourmash/blob/c176cd27f349cfda9e5d2061b100b483804ef4a8/setup.py#L69